backend/src/src-cvesearch/src/main/java/org/eclipse/sw360/cvesearch/datasource/heuristics/SearchLevels.java

```diff
@@ -172,54 +158,9 @@ public SearchLevels addGuessingSearchLevels(CveSearchApi cveSearchApi, int vendo
                 .collect(Collectors.toList());
     }
 
-    //==================================================================================================================
-    protected Function<Release,String> escapeGeneratorResult(Function<Release,String> generator) {
-        return r -> generator.apply(r)
-                .replace('/','.')
-                .replace('\\','.')
-                .replace('*','.')
-                .replace('+','.')
-                .replace('!','.')
-                .replace('?','.')
-                .replace('^','.')
-                .replace('$','.')
-                .replace('[','.')
-                .replace(']','.')
-                .replace(" ", CPE_WILDCARD)
-                .toLowerCase();
-    }
-
-    protected Function<Release, String> implodeSearchNeedleGenerators(Function<Release,String> generator, Function<Release,String> ... generators){
-        return Arrays.stream(generators)
-                .map(this::escapeGeneratorResult)
-                .reduce(generator,
-                        (g1,g2) -> r -> g1.apply(r) + CPE_WILDCARD + g2.apply(r));
-    }
-
     protected boolean isCpe(String potentialCpe){
         return (! (null == potentialCpe))
-                && potentialCpe.startsWith("cpe:")
+                && ( potentialCpe.startsWith("cpe:2.3:") || potentialCpe.startsWith("cpe:/") )
```

Comment:
Would you be willing to define constants for both old and new cpe prefixes? They are used in several places throughout the code. I think creating named constants is justified.